lib/gcloud/datastore/dataset.rb

```diff
@@ -321,6 +319,133 @@ def transaction
         end
       end
 
+      ##
+      # Create a new Query instance. This is a convenience method to make the
+      # creation of Query objects easier.
+      #
+      # === Parameters
+      #
+      # +kinds+::
+      #   The kind of entities to query. This is optional. (+String+)
```

Comment:
Doc is for singular kind, but parameter supports multiple/list?
"kind" -> "kind or kinds"
String -> String or multiple